packages/flutter/test/material/dropdown_form_field_test.dart

```diff
@@ -195,6 +195,41 @@ void main() {
     expect(value, equals('three'));
   });
 
+  testWidgets('DropdownButtonFormFieldState value changes when option selected',
+      (WidgetTester tester) async {
+    String value = 'one';
+    final GlobalKey<FormFieldState<String>> stateKey =
+        GlobalKey<FormFieldState<String>>();
+
+    await tester.pumpWidget(
+      Builder(
+        builder: (BuildContext context) {
+          return MaterialApp(
+            home: Material(
+              child: DropdownButtonFormField<String>(
+                key: stateKey,
+                value: value,
+                items: menuItems.map((String value) {
```

Comment:
nit: typically the child[ren] widget[s] are at the end of the argument list. In this case, you could put `items` at the end of the argument list.